src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
Your implementation won’t allow us to use polymorphism in the code. Your reference is bonded to specific implementation, but it is always better to depend on the abstraction. 
Machine excavator = new Excavator();